checker/manual/creating-a-checker.tex

```diff
@@ -1109,34 +1107,121 @@ \section{The checker class:  Compiler interface\label{writing-compiler-interface
 checker classes for the Interning Checker and the Nullness Checker:
 
 \begin{Verbatim}
-  @TypeQualifiers({ Interned.class, PolyInterned.class, PolyAll.class })
   @SupportedLintOptions({"dotequals"})
   public final class InterningChecker extends BaseTypeChecker { }
 
-  @TypeQualifiers({ Nullable.class, Raw.class, NonNull.class, PolyNull.class, PolyAll.class })
   @SupportedLintOptions({"flow", "cast", "cast:redundant"})
   public class NullnessChecker extends BaseTypeChecker { }
 \end{Verbatim}
 
+\subsection{Indicating supported annotations\label{indicating-supported-annotations}}
+
+A checker must indicate the annotations that it supports (make up its type
+hierarchy), including whether it supports the polymorphic qualifier
+\refqualclass{framework/qual}{PolyAll}.
+
+By default, a checker supports \<PolyAll>, and all annotations located in a
+subdirectory called \<qual> that's located in the same directory as the checker.
```

Comment:
By directory do you really mean package? Looking at the code, I can't tell if the directory structure of the Java files must match the package structure.  (If it does, that's fine -- no need to change anything.  I just want to make sure this is accurate.)